### ISSUE TITLE
Date picker is not accessible in IPhone with voiceover

### ISSUE THREAD

**Issue by ranag14** [telerik/kendo-angular#1166]
Hi,
I'm not sure how to use date picker in IPhone with voiceover. Once user tabs into the control & does double tab then what's after that. How user with disability will fill the date control.

**Comment by ggkrustev** [comment]
Hi @ranag14,

The VoiceOver reader has dedicated keyboard keys combination that it enforces if the user wants to interact with the component. On the other hand, the DateInput, used internally in the DatePicker, uses just UP/DOWN arrows to increase/decrease the value. That being said, the user can easily modify the value just using either the UP or DOWN arrow.

We will further investigate the case and will try to find a better solution for the described scenario. For the time being, please inform the users that they need to use the simplified keyboard keys combination.
